modules/admin_manual/pages/configuration/files/encryption/index.adoc

```diff
@@ -114,7 +114,6 @@ The following files are *never* encrypted:
 +
 NOTE: You can post encrypt existing files via an xref:configuration/server/occ_command.adoc#encryption[occ encryption command].
 * Existing files in Version
-* Image thumbnails from the Gallery app
```

Comment:
Image thumbnails are independent of the Gallery app. I think I wrote this in another PR already: Please just remove "from the Gallery app".

Question: Why is this same text written in three different files?

Reply:
I'll move the repeated text to a separate, includable, partial.